src/psyclone/nemo.py

```diff
@@ -439,7 +425,19 @@ def get_kernel_schedule(self):
         :returns: the kernel schedule representing the inlined kernel code.
         :rtype: :py:class:`psyclone.psyGen.KernelSchedule`
         '''
-        return self._kern_schedule
+        return self._children[0]
+
+    def node_str(self, colour=True):
```

Comment:
I don't think you need this method as it's doing exactly the same as the one in the `Node` base class?

Reply:
The function in the base class uses self._arguments, which is not defined for an Inlined (or Nemo) kern. While I could of course add a method to inlined class that returns an empty string as arguments, I think it would be better for future refactoring not to do this.

Reply:
Oh yes, my mistake.